### PR TITLE
Check filter selected and show selected items on top

### PR DESCRIPTION
**Pull request by Letrab**
Issue #39

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -304,14 +304,15 @@ module.exports.buckets = function(items, field, agg, aggregations) {
   buckets = _.map(buckets, (val, key) => {
     return {
       key: key,
-      doc_count: val
+      doc_count: val,
+      selected: helpers.includes_any_element(key, agg.filters)
```
Comment (cigolpl) [inline src/lib.js]: IMO `_.includes(key, agg.filters)` will be easier and maybe faster

File: tests/aggregationsSpec.js [hunk 2]
```diff
@@ -22,6 +22,142 @@ describe('aggregations', function() {
     actors: ['e']
   }]
 
+  var moreItems = [{
```
Comment (cigolpl) [inline tests/aggregationsSpec.js]: That json array should go to './fixtures/movies_some_name.json' and be loaded here like `var moreItems = require('./fixtures/movies_some_name.json')`

**Comment by cigolpl** [comment]
Looks good!

**Comment by Letrab** [comment]
Nice! Thank you very much for your quick responses and integration!

**Pull request merged**